.wrgl/config.yaml

```diff
@@ -1848,6 +1844,12 @@ branch:
     file: data/fuse/per_rayne_pd.csv
     primaryKey:
       - uid
+  citizen-new-orleans-pd:
+    remote: citizen-new-orleans-pd
+    merge: refs/heads/main
+    file: data/fuse/citizen_new_orleans_pd.csv
+    primaryKey:
+      - citizen_uid
```

Comment:
You should introduce new tables called `uof_citizens` and `uof_officers` here as well, and they are the merged results of data from multiple datasets, not just from New Orleans. It's a pain but we might as well make the changes now to make things consistent.